jitpack.yml

```diff
@@ -0,0 +1,12 @@
+before_install:
```

Comment:
can be removed, right?

Reply:
Or we leave it, so it would theoretically work with jitpack? But deleting a file also feels good :)